src/assets/wise5/authoringTool/add-branch/add-branch.component.ts

```diff
@@ -225,21 +233,27 @@ export class AddBranchComponent {
     return this.formGroup.get('mergeStep').value;
   }
 
-  protected submit(): void {
-    const data: any = {};
-    data.branchStepId = this.targetId;
-    data.pathCount = this.getPathCount();
-    data.criteria = this.getCriteria();
-    data.nodeId = this.getNodeId();
-    data.componentId = this.getComponentId();
+  private getBranchParams(): CreateBranchParams {
+    const data: any = {
```

Comment:
Rename to params and give CreateBranchParams type?
```suggestion
    const params: CreateBranchParams = {
```